internal/cli/kraft/kraft.go

```diff
@@ -163,6 +163,20 @@ func Main(args []string) int {
 		ctx = iostreams.WithIOStreams(ctx, copts.IOStreams)
 	}
 
+	if (os.Getenv("SUDO_UID") != "" || os.Getenv("SUDO_GID") != "" || os.Getenv("SUDO_USER") != "") && os.Getenv("KRAFTKIT_NO_WARN_SUDO") == "" {
```

Comment:
can you switch this to a config kraftkit flag so we can then also use it as a flag `--no-warn-sudo`?